src/dftbp/dftb/periodic.F90

```diff
@@ -874,7 +875,7 @@ subroutine fillNeighbourArrays(neigh, iNeighbour, neighDist2, startAtom, endAtom
     integer :: maxNeighbourLocal
   #:endif
 
-    if (isParallel) then
+    if (neigh%useMpiWindows_) then
```

Comment:
If `isParallel` is indeed not required, we could also remove it from the routine's signature.